spec/commands/decidim/create_registration_spec.rb

```diff
@@ -0,0 +1,263 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Comments
+    describe CreateRegistration do
+      describe "call without registration_fields" do
+        # Create an organisation with registration fields disabled
+        let(:organization) { create(:organization, registration_fields: { "enabled" => false }) }
```

Comment:
Prefer to use available factory trait 

```suggestion
        let(:organization) { create(:organization, :registration_fields_disabled) }
```